src/imio/smartweb/core/vocabularies.py

```diff
@@ -251,3 +256,55 @@ def __call__(self, context=None):
 
 
 ImageSizeVocabulary = ImageSizeVocabularyFactory()
+
+
+class ConcatCategoryTopicsVocabularyFactory:
```

Comment:
This name is not explicit enougth `CategoryAndTopicsVocabularyFactory` is better